packages/components/inputs/src/controlled-input/ControlledInput.tsx

```diff
@@ -128,12 +138,9 @@ const _ControlledInput = (
       />
       {type === 'radio' ? (
         // eslint-disable-next-line jsx-a11y/label-has-associated-control
-        <label
-          className={cx(styles.ghost, styles.ghostRadioButton)}
-          htmlFor={id}
-        />
+        <label {...labelProps} className={labelClassnames} htmlFor={id} />
```

Comment:
Shouldn't this use the Label component?

Reply:
Good point, I didn't know about Label when I was migrating the component :)

Reply:
I tried but
```
export type LabelInternalProps = {
  /**
   * Label value to show
   */
  children: React.ReactText;
  ...
}
```

It doesn't allow me to pass actual components. I could only do it for the first of both cases which doesn't have children at all. But I need both to do the same so, `Label` doesn't work here :/

Reply:
Ah we should just change the type then, that was just me trying to narrow it down. Of course it should allow for icons.